src/imitation/scripts/common/expert.py

```diff
@@ -43,3 +43,19 @@ def config_hook(config, command_name, logger):
 @expert_ingredient.capture
 def get_expert_policy(venv, policy_type, loader_kwargs, common):
     return serialize.load_policy(policy_type, venv, **loader_kwargs)
+
+
+@expert_ingredient.capture
+def download_expert_rollouts(rollout_path, loader_kwargs):
+    if not rollout_path.endswith("-huggingface"):
+        raise ValueError(
+            "`rollout_path` must follow the convention `{algo}-huggingface`"
```

Comment:
Ironically, it seems like you're following a different convention check on `demonstrations.py`?